amstrax/plugins/events/event_coincidences.py

```diff
@@ -0,0 +1,105 @@
+import numba
+import numpy as np
+import amstrax
+import strax
+
+export, __all__ = strax.exporter()
+
+
+@export
+@strax.takes_config(
+        strax.Option(
+        "max_delay",
+        default=100,
+        help="Maximum allowed time difference between XAMS events and external peaks to count as a match, in ns",
+        ),
+        strax.Option(
+        "na_peak_delta",
+        default=65,
+        help="Sets the window as [511 - delta, 511 + delta] for the allowed energies that count as a 511 keV photon in the external detector",
```

Comment:
Maybe make a little bit more clear that this is in units of keV. Suggestion: change name of ``na_peak_delta`` to ``absorption_peak_delta`` (or similar)